src/metabase/models/user.clj

```diff
@@ -287,6 +287,18 @@
    (schema/optional-key :google_auth)      schema/Bool
    (schema/optional-key :ldap_auth)        schema/Bool})
 
+(def HydratedUser
+  "Standard form of a user (for consumption by the frontend and such)"
+  {:id           su/IntGreaterThanOrEqualToZero
+   :email        su/NonBlankString
+   :first_name   su/NonBlankString
+   :last_name    su/NonBlankString
+   :common_name  su/NonBlankString
+   :last_login   schema/Any
+   :date_joined  schema/Any
+   :is_qbnewb    schema/Bool
+   :is_superuser schema/Bool})
```

Comment:
Renamed to DefaultUser, but (simple) code-sharing with `default-user-columns` doesn't work since the schema does contain a hydrated value: `common_name`

Reply:
s/hydrated/post-selected